BFSTree.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+class TreeNode3
```

Comment:
Please rename this to something commonly being used. Since this repo will be used to have all the algorithms related to java.

You can rename this class to `TreeNode3`.

Reply:
Thank you for the suggestion really appreciated 😀

Reply:
i have renamed it to class TreeNode